mongo_connector/connector.py

```diff
@@ -1189,7 +1189,12 @@ def log_startup_info():
                     __version__)
     LOG.always('Python version: %s', sys.version)
     LOG.always('Platform: %s', platform.platform())
-    LOG.always('pymongo version: %s', pymongo.__version__)
+    pymongo_version = 'unknown'
```

Comment:
'version' has existed for forever (and still exists today), so this all can be simplified to:

```
if hasattr(pymongo, '__version__'):
    pymongo_version = pymongo.__version__
else:
    pymongo_version = pymongo.version
```

You could also just use 'version', but I guess checking for __version__ is a bit more future proof.